code/API_definitions/device-reachability-status-subscriptions.yaml

```diff
@@ -36,7 +36,11 @@ info:
 
     Note: Additionally to these list, ``org.camaraproject.reachability-status-subscriptions.v0.subscription-ends`` notification `type` is sent when the subscription ends.
     This notification does not require dedicated subscription.
-    It is used when the subscription expire time (optionally set by the requester) has been reached or if the API server has to stop sending notification prematurely.
+
+    It is used when:
+      - the subscription expire time (optionally by the requester) has been reached
```

Comment:
Maybe better: "It is used in following cases:" to make it clear that the cases are not AND linked